pyakeneo/result.py

```diff
@@ -74,23 +88,46 @@ def get_self_link(self):
     def get_first_link(self):
         return self._link_first
 
-    @staticmethod
-    def parse_page(json_data):
+    @classmethod
```

Comment:
Why this change if you don't use anything from `cls`?

Reply:
In case some day someone decides to inherit this class and customize that method

Reply:
You can override a staticmethod too, but nevermind